$NetBSD: patch-configure,v 1.1 2012/02/07 11:34:00 fhajny Exp $

* Fix DTrace flags.
* Avoid installing under ${PREFIX}/lib/amd64 on SunOS.

--- configure.orig	2011-09-28 15:26:49.000000000 +0000
+++ configure
@@ -5267,10 +5267,11 @@ fi
 
 
     isainfo_k=`${ISAINFO} -k`
-    DTRACEFLAGS="${DTRACEFLAGS} -${isainfo_b}"
+    DTRACEFLAGS="${DTRACEFLAGS}"
 
     if test "x$ac_enable_64bit" = "xyes"; then :
 
+      DTRACEFLAGS="${DTRACEFLAGS} -64"
 
       if test "x${ac_cv_env_LDFLAGS_set}" = "x"; then :
 
@@ -5278,11 +5279,6 @@ fi
 
 fi
 
-      if test "x$libdir" = "x\${exec_prefix}/lib"; then :
-
-                     libdir="${libdir}/${isainfo_k}"
-
-fi
 
       if test "x${ac_cv_env_CFLAGS_set}" = "x"; then :
 
